hyperstart/api/grpc/hyperstart.proto

```diff
@@ -55,6 +57,14 @@ message WaitProcessResponse {
 	int32 status = 1;
 }
 
+message StopContainerRequest {
+	string container_id = 1;
+}
```

Comment:
You probably want a timeout here as well.

Reply:
I think StopContainer should force kill the container if agent fails to stop it with the stop signal. Is the timeout used to determine when agent should think stop fails and initiate SIGKILL?

Reply:
@sameo I am okay with adding a timeout here.
@bergwolf I think the timeout would be about StopContainer returning an error if we could not stop the container. I would use SIGKILL from the beginning with StopContainer.

Reply:
@sboeuf SIGTERM gives container apps a chance to gracefully stop before being killed by SIGKILL. This is also how `docker stop` works. I think we should follow the behavior if we introduce StopContainer API.